use-timescale/continuous-aggregates/about-continuous-aggregates.md

```diff
@@ -17,6 +17,86 @@ import CaggsTypes from "versionContent/_partials/_caggs-types.mdx";
 
 <CaggsTypes />
 
+## Components of a continuous aggregate
+
+Continuous aggregates consist of:
+
+*   Materialization hypertable to store the aggregated data in
+*   Materialization engine to aggregate data from the raw, underlying, table to
+    the materialization hypertable
+*   Invalidation engine to determine when data needs to be re-materialized, due
+    to changes in the data
+*   Query engine to access the aggregated data
+*   <Optional/>A refresh policy, used to keep the continuous aggregate updated
+    in the background.
+
+### Materialization hypertable
+
+Continuous aggregates take raw data from the original hypertable, aggregates it,
+and store the intermediate state in a materialization hypertable. When you query
+the continuous aggregate view, the state is returned to you as needed.
+
+The materialization table is stored as a hypertable, to take advantage of the
```

Comment:
Unfortunately, this isn't true at all, still. While we have that kind of information internally, it isn't part of the actual continuous aggregate.

```
CREATE MATERIALIZE VIEW xyz
  WITH (timescaledb.continuous) AS
SELECT
    time_bucket('1 day', ts) bucket,
    avg(value) average
FROM some_hypertable
```

Will create a materialization hypertable that looks exactly like what it's described as above:
```
| bucket timestamptz | average float8 |
```

No partial or intermediate state is stored here. If you ask for a `counter_agg` column, the 2-step aggregation counter_agg specifically will create some intermediate state and store it, but that is part of how the aggregation is implemented, not part of the continuous aggregate.

If you want an average which can be re-aggregated, you have to store sum and count separately as in:
```
CREATE MATERIALIZE VIEW xyz
  WITH (timescaledb.continuous) AS
SELECT
    time_bucket('1 day', ts) bucket,
    avg(value) average,
    count(*) intermediate_count,
    sum(value) intermediate_sum
FROM some_hypertable
```